Userland/Applications/FileManager/main.cpp

```diff
@@ -464,13 +464,13 @@ ErrorOr<int> run_in_desktop_mode()
     auto open_terminal_action = GUI::Action::create("Open in &Terminal", {}, TRY(Gfx::Bitmap::load_from_file("/res/icons/16x16/app-terminal.png"sv)), [&](auto&) {
         auto paths = directory_view->selected_file_paths();
         if (paths.is_empty()) {
-            spawn_terminal(directory_view->path());
+            spawn_terminal(window.ptr(), directory_view->path());
```

Comment:
Couldn't we take advantage of `operator T*()` and discard `.ptr()`?
(There are also two more occurrences in the commit)

Reply:
Didn't know we had that operator :sweat_smile: